src/_includes/tools.include.liquid

```diff
@@ -0,0 +1,22 @@
+<div class="container">
+  <div>
+    {% if tools %}
```

Comment:
why this if-check? shouldn't we just not load the whole include if there is no tools defined ?

Reply:
Yeah that is from the previous setup. I'll drop it.